.github/workflows/release.yml

```diff
@@ -60,12 +37,7 @@ jobs:
       run: poetry install
 
     - name: Run pytest
-      run: poetry run coverage run --source ./src/grpc_argument_validator -m unittest discover -s src/tests -p '*_test.py'
-
-    - name: Upload Coverage Report
-      run: |
-        poetry run coverage xml --fail-under=0
-        poetry run codecov
+      run: poetry run python -m unittest discover -s src/tests -p '*_test.py'
```

Comment:
Is this job still necessary in the release workflow?

Reply:
The test badge will be named using `<workflow>.name`, which is why I separated `Tests` from `Release` in the first place. 

It is AFAIK impossible to let one workflow depend on another, so these tests are there to guarantee that we cannot release a package for which tests fail. So I agree, that it seems redundant to have tests here, but I cannot find a better alternative as of now.

Reply:
Cool :)